docs/pages/reference/helm-reference/teleport-kube-agent.mdx

```diff
@@ -655,6 +657,49 @@ You can specify multiple selectors by adding elements to the list.
   Once `databaseResources` is set, you can dynamically register database with `tsh` by following [this guide](../../database-access/guides/dynamic-registration.mdx).
 </Admonition>
 
+
+## kubernetesDiscovery
+
+| Type   | Default value | Required?                                                                                                                             |
+|--------|---------------|---------------------------------------------------------------------------------------------------------------------------------------|
+| `list` | `See below`   | When the `discovery` chart role is used, `kubernetesDiscovery` is required. |
```

Comment:
```suggestion
| `list` | Read access to all applications in all cluster namespaces.   | When the `discovery` chart role is used, `kubernetesDiscovery` is required. |
```
I would change `See below` to a description of the default access permissions. Otherwise, (a) `See below` seems like a default value itself, which is a bit unexpected and (b) it's not very explicit where a reader should look.

Feel free to change my suggested replacement to something more accurate!